iOS-tic-tac-toe/BoardStackView.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+import UIKit
+
+public class BoardStackView: UIStackView, BoardView {
+    
+    public func show(board board: [String]) {
+        for view in self.subviews{
+            for button in view.subviews {
+                let btn = button as! UIButton
+                if let image = UIImage(named: "letter-\(board[btn.tag].lowercaseString)") {
+                    btn.enabled = false
+                    btn.setBackgroundImage(scaleImage(image, button: btn), forState: .Normal)
+                }
+            }
+        }
+    }
+    
+    public func spotsEnabled(enabled: Bool) {
```

Comment:
The name of this function reads a bit strangely to me. It's not clear to me what this is doing based on the name alone.

The `enabled: Bool` parameter to me indicates that this method has two different ways of being called, which makes me wonder if it should be two separate functions.

I would consider either renaming the function or creating two separate functions. Something like `disableInput` and `enabledInput`, however I would leave the exact names up to you